analytics/views.py

```diff
@@ -1132,6 +1134,16 @@ def support(request: HttpRequest) -> HttpResponse:
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
             context["message"] = msg
+        elif request.POST.get("new_subdomain", None) is not None:
+            new_subdomain = request.POST.get("new_subdomain")
+            old_subdomain = realm.string_id
+            if not Realm.objects.filter(string_id__exact=new_subdomain).exists():
+                do_change_realm_subdomain(realm, new_subdomain)
+                msg = f"Subdomain changed from {old_subdomain} to {new_subdomain}"
+                context["message"] = msg
```

Comment:
You don't need the temporary msg variable. You can directly assign the message to `context["message"]`. It was used in the other cases to prevent the line from getting too big